packages/nextjs/src/server/index.ts

```diff
@@ -24,13 +24,12 @@ const globalWithInjectedValues = global as typeof global & {
 
 const domain = domainModule as typeof domainModule & { active: (domainModule.Domain & Carrier) | null };
 
-// Exporting this constant means we can compute it without the linter complaining, even if we stop directly using it in
-// this file. It's important that it be computed as early as possible, because one of its indicators is seeing 'build'
-// (as in the CLI command `next build`) in `process.argv`. Later on in the build process, everything's been spun out
-// into child threads and `argv` turns into ['node', 'path/to/childProcess.js'], so the original indicator is lost. We
-// thus want to compute it as soon as the SDK is loaded for the first time, which is normally when the user imports
-// `withSentryConfig` into `next.config.js`.
```

Comment:
This comment was only applicable back when we had `instrumentServer` which we recently removed.